scanner/families/malware/clam/clam.go

```diff
@@ -176,7 +183,7 @@ func (s *Scanner) runClamScanWithDaemon(ctx context.Context, fsScanPath string)
 		/* If the error is that malware was found, this is not an actual error, Clamd returns
 		   a non 0 exit code when malware was found */
 		var runError utils.CmdRunError
-		if !errors.As(err, &runError) || !strings.Contains(string(runError.Stdout), constants.MalwareDetectedIndication) {
+		if !errors.As(err, &runError) || !strings.Contains(string(runError.Stdout), constants.ScanSummaryText) {
```

Comment:
please note that we don't display summary for clamdscan (the `--no-summary` flag is passed to not display), so we would need to remove the flag to avoid erroring here

Did you have a chance to test both clamscan and clamdscan scanners with a a) no malwares and b) one specific malware?

Reply:
Thanks, I've fixed it.

I've tested clamscan and clamdscan scanning with no malwares and with the EICAR test file.